src/foam/nanos/auth/User.js

```diff
@@ -660,3 +650,44 @@ foam.CLASS({
     }
   ]
 });
+
+foam.CLASS({
+  package: 'foam.nanos.auth',
+  name: 'FormattedPhoneNumber',
+  extends: 'foam.core.PhoneNumber',
```

Comment:
We should refine `foam.core.PhoneNumber` rather than extend it. I don't see any reason `PhoneNumber` itself shouldn't have these validation predicate.

```suggestion
  refines: 'foam.core.PhoneNumber',
```

This refinement should also move to `Validation.js` so it can be found with the rest of the validation refinements.

Reply:
Yes, a refinement, sure.
I also thought integrating them directly to phone number but thought maybe other validation rules in other countries may exist, I don't know.